src/docs/build-and-earn/testnet-guides/prerequisites.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Prerequisites and Operating Systems
+---
+
+# ﻿Prerequisites and Operating Systems
+
+You can run your node in different operating systems such as Debian 10, Ubuntu, WSL, and macOS Catalina version 10.15.4 or above. If you are installing docker for the first time, please check the next section, “Notes for Installing Docker for the First Time”.
+
+The first step is to ensure that you have set up all requirements and prerequisites as the following:
+
+- Install [Docker](https://docs.docker.com/engine/install/) 20.10.6 or above
+- Install [Docker Compose](https://docs.docker.com/compose/install/) 1.29.1 or above
+- Install [Git](https://github.com/git-guides/install-git)
+- Install a code editor such as Nano, Atom, etc.
+
+Missing any of the prerequisites above can lead to errors and being unable to run your node. It is recommended to have the latest version of Docker and Docker compose installed. If you already installed them, you can check the version by running the following commands in your terminal:
+```
+docker -v
+docker-compose -v
+```
+
+## Setup the environment before running a node
+
+After installing all prerequisites, you can start to set up the environment:
+1. Create a directory to use as your workspace:
+```
+mkdir ~/cudos
+cd ~/cudos
+```
+2. Make sure that you are in the correct directory (cudos directory in this example)
+3. Clone the correct branches from the [CudosNode](https://github.com/CudoVentures/cudos-node), [CudosBuilders](https://github.com/CudoVentures/cudos-builders), and [CudosGravityBridge](https://github.com/CudoVentures/cosmos-gravity-bridge) repositories:
+```
+git clone --depth 1 -b testnet https://github.com/CudoVentures/cudos-node.git CudosNode
+git clone --depth 1 -b master  https://github.com/CudoVentures/cudos-builders.git CudosBuilders
+git clone --depth 1 -b master https://github.com/CudoVentures/cosmos-gravity-bridge.git CudosGravityBridge
+```
+4. Change to the root of each local repository and check out the following branches:
+```
+cd CudosNode
+git checkout gravity/sdk-0.43
+
+cd..
+cd CudosBuilders
+git checkout sdk-0.43
+
+cd..
+cd CudosGravityBridge
+git checkout cosmos-sdk-0.43
+```
+5. Put all cloned repositories in the same directory
+6. Rename the folders accordingly to exactly _CudosNode_, _CudosBuilders_, and _CudosGravityBridge_
+7. Navigate to the _CudosBuilders_ directory
+8. Build the docker image of the binary by running the command:
+```
+cd ./docker/binary-builder&& sudo docker-compose --env-file ./binary-builder.arg -f ./binary-builder.yml -p cudos-binary-builder build
+```
+
+Note that you need to repeat the same steps for each node that you want to create.
```

Comment:
potential improvement (for the future): as @peterwillis suggested, this could be automatically built (once) by the ci/cd pipeline and users just pointed to download the corresponding binary for their os

the time needed for these builds (on my machine) is not that insignificant (x3 for all nodes!):
```
real    3m34.236s
user    0m0.726s
sys     0m0.204s
```